pkg/cmd/experimental/project/new_project.go

```diff
@@ -0,0 +1,118 @@
+	projectName  string
+	displayName  string
+	description  string
+	clientConfig clientcmd.ClientConfig
+
+	adminRole             string
+	masterPolicyNamespace string
+	adminUser             string
+}
+
+func NewCmdNewProject(name string) *cobra.Command {
+	options := &newProjectOptions{}
+
+	cmd := &cobra.Command{
+		Use:   name + " <project-name>",
+		Short: "create a new project",
+		Long:  `create a new project`,
+		Run: func(cmd *cobra.Command, args []string) {
+			if !options.complete(cmd) {
+				return
+			}
+
+			err := options.run()
+			if err != nil {
+				fmt.Printf("%v\n", err)
+			}
+		},
+	}
+
+	// Override global default to https and port 8443
+	clientcmd.DefaultCluster.Server = "https://localhost:8443"
+	clientConfig := cmdutil.DefaultClientConfig(cmd.Flags())
+	options.clientConfig = clientConfig
+
+	// TODO remove once we have global policy objects
+	cmd.Flags().StringVar(&options.masterPolicyNamespace, "master-policy-namespace", "master", "master policy namespace")
+	cmd.Flags().StringVar(&options.adminRole, "admin-role", "admin", "project admin role name in the master policy namespace")
+	cmd.Flags().StringVar(&options.adminUser, "admin", "", "project admin username")
+	cmd.Flags().StringVar(&options.displayName, "display-name", "", "project display name")
+	cmd.Flags().StringVar(&options.description, "description", "", "project description")
+
+	return cmd
+}
+
+func (o *newProjectOptions) complete(cmd *cobra.Command) bool {
+	args := cmd.Flags().Args()
+	if len(args) != 1 {
+		cmd.Help()
+		return false
+	}
+
+	o.projectName = args[0]
+
+	return true
+}
+
+func (o *newProjectOptions) run() error {
+	clientConfig, err := o.clientConfig.ClientConfig()
+	if err != nil {
+		return err
+	}
+	client, err := client.New(clientConfig)
+	if err != nil {
+		return err
+	}
+
+	_, err = client.Projects().Get(o.projectName)
+	projectFound := !kerrors.IsNotFound(err)
+	if (err != nil) && (projectFound) {
+		return err
+	}
+	if projectFound {
+		return fmt.Errorf("project %v already exists", o.projectName)
+	}
+
+	project := &projectapi.Project{}
+	project.Name = o.projectName
+	project.DisplayName = o.displayName
+	project.Annotations = make(map[string]string)
+	project.Annotations["description"] = o.description
+	project, err = client.Projects().Create(project)
+	if err != nil {
+		return err
+	}
+
+	if len(o.adminUser) != 0 {
+		adminRoleBinding := &authorizationapi.RoleBinding{}
+
+		adminRoleBinding.Name = "admins"
+		adminRoleBinding.RoleRef.Namespace = o.masterPolicyNamespace
+		adminRoleBinding.RoleRef.Name = o.adminRole
+		adminRoleBinding.UserNames = []string{o.adminUser}
+
+		_, err := client.RoleBindings(project.Name).Create(adminRoleBinding)
+		if err != nil {
+			fmt.Printf("Unable to add %v as an admin on project %v, but the project was created.  Try running `openshift ex policy add-user --namespace=%v --role-namespace=%v %v %v`\n", o.adminUser, o.projectName, o.projectName, o.masterPolicyNamespace, o.adminRole, o.adminUser)
```

Comment:
nits:

1. put success first, then failure
2. sub in the specified role rather than hardcoding admin
3. put sample command on it's own line to make it easier to to copy/paste

```
fmt.Printf("The project %v was created, but %v could not be added to the %v role.\n", ...)
fmt.Println("To add the user to the existing project, run\n\n\topenshift ex policy add-user --namespace=%v --role-namespace=%v %v %v\n", ...)
```